cirq/__init__.py

```diff
@@ -160,6 +151,19 @@
     ZPowGate,
     ZZ,
     ZZPowGate,
+    amplitude_damp,
```

Comment:
Why'd these move to the end? Because lower case?

Reply:
Yeah. `ord('a') > ord('Z')`.